### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by haoyuying** [baowenbo/DAIN#94]
the program is broken when the program excutes "y_s,offset,filter = model(torch.stack((X0, X1),dim = 0))", I have pointed out align_corners "temps = [nn.Upsample(scale_factor=4, mode='bilinear',align_corners=False)(temp)  for temp in temps]", however it does not work.

/mnt/haoyuying/anaconda3/envs/pytorch-env/lib/python3.7/site-packages/torch/nn/modules/upsampling.py:129: UserWarning: nn.UpsamplingNearest2d is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.{} is deprecated. Use nn.functional.interpolate instead.".format(self.name))
/mnt/haoyuying/anaconda3/envs/pytorch-env/lib/python3.7/site-packages/torch/nn/modules/upsampling.py:129: UserWarning: nn.Upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.{} is deprecated. Use nn.functional.interpolate instead.".format(self.name))
/mnt/haoyuying/anaconda3/envs/pytorch-env/lib/python3.7/site-packages/torch/nn/functional.py:2423: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
Segmentation fault (core dumped)
